docs/content/latest/yugabyte-platform/install-yugabyte-platform/prepare-environment/aws.md

```diff
@@ -135,25 +135,60 @@ In order for Yugabyte Platform to manage YugabyteDB nodes, limited access to you
+  Ensure that **Auto-assign Public IP** is enabled (if it is disabled, the instance would not be accessible from outside AWS). 
 
-- Add a tag to name the machine. You can set key to `Name` and value to `yugaware-1`. Continue to the next step.
+  If you created an IAM role, as described in [Create an IAM role](#create-an-iam-role-(optional)), or already had the IAM role that you would like to use, include this information under **IAM role**. See [Deploy the YugabyteDB universe using an IAM role](#deploy-the-yugabytedb-universe-using-an-iam-role) for more information.
 
-- Select the `yugaware-sg` security group created in the previous step (or the custom name you chose when setting up the security groups). Launch the instance.
+- Increase the root storage volume size to at least 100GiB.
 
-- Pick an existing key pair (or create a new one) in order to access the machine. Make sure you have the ssh access key. This is important to enable `ssh` access to this machine. In this example, assume that the key pair is `~/.ssh/yugaware.pem`.
+- Add a tag to name the instance. You can set key to `Name` and value to `yugaware-1`.
 
-Finally, click **Launch** to launch the Yugabyte Platform server. You should see a machine being created as shown in the image below.
+- Select the `yugaware-sg` security group created in the previous step (or the custom name you chose when setting up the security groups), and then launch the instance.
 
-![Pick OS Image](/images/ee/aws-setup/yugaware-machine-creation.png)
+- Pick an existing key pair or create a new one in order to access the instance. 
+
+  Ensure that you have the SSH access key because it is required for enabling `ssh` access to the computer. In this example, assume that the key pair is `~/.ssh/yugaware.pem`.
+
+- Click **Launch** to launch the Yugabyte Platform server. 
+
+  You should see an instance being created, as shown in the following illustration:<br><br>
+
+  ![Image](/images/ee/aws-setup/yugaware-machine-creation.png)
+
+### Deploy the YugabyteDB universe using an IAM role
+
+If you are planning to use an IAM role while deploying the universe in your AWS account, you need to attach the IAM role to the Yugabyte Platform VM before launching the instance, as follows:
+
+1. Navigate to **EC2 > New Instance > Confirm Instance Details > Attach the IAM role**.
+2. Set the **IAM role** field to ec2-admin-access.
+3. Set the **Metadata accessible** field to Enabled.
+4. Set the **Metadata version** field to V1 and V2 (token optional).
+5. Set the **Metadata token response hop limit** field to 3, as per the following illustration:
+
+  ![AIM for AWS](/images/ee/aws-setup/iam-for-aws.png)
+
+If you are configuring an existing instance of Yugabyte Platform, execute the following command (replace `NNNNNNN` with the instance ID):
+
+```shell
+aws ec2 modify-instance-metadata-options --instance-id i-NNNNNNN --http-put-response-hop-limit 3 --http-endpoint enabled --region us-west-2
```

Comment:
`--region us-west-2` --> maybe make it a bit more clear that the region here is just shown as an example + that it will depend on which region this ec2 VM is deployed in.

Reply:
Changed as  ... (replace `NNNNNNN` with the instance ID and `us-west-2` with the region in which this EC2 VM is deployed):